go/cmd/dolt/commands/push.go

```diff
@@ -199,10 +199,7 @@ func getDefaultRemote(sqlCtx *sql.Context, queryist cli.Queryist) (string, error
 			return "origin", nil
 		}
 	}
-	for _, row := range rows {
-		return row[0].(string), nil
-	}
-	return "", env.ErrCantDetermineDefault
+	return rows[0][0].(string), nil
```

Comment:
I think there is  still a need for the error, yes? If there are no rows then you'll get an index out of bounds error here.